src/library/scala/collection/immutable/ChampHashMap.scala

```diff
@@ -233,6 +239,24 @@ private final class BitmapIndexedMapNode[K, +V](val dataMap: Int, val nodeMap: I
   def getNode(index: Int) =
     content(content.length - 1 - index).asInstanceOf[MapNode[K, V]]
 
+  def apply(key: K, originalHash: Int, keyHash: Int, shift: Int): V = {
+    val mask = maskFrom(keyHash, shift)
+    val bitpos = bitposFrom(mask)
+
+    if ((dataMap & bitpos) != 0) {
+      val index = indexFrom(dataMap, mask, bitpos)
+      val key0 = this.getKey(index)
+      return if (key == key0) this.getValue(index) else throw new NoSuchElementException
+    }
+
+    if ((nodeMap & bitpos) != 0) {
+      val index = indexFrom(nodeMap, mask, bitpos)
+      return this.getNode(index).get(key, originalHash, keyHash, shift + BitPartitionSize)
```

Comment:
Shouldn't this be `apply` instead of `get`? How does it even compile?

Reply:
You're right. This was being typecheked as `.get.apply(...)`. The `.get` method is detritus from an earlier design. I'll remove it and replace this with `.apply`.